python/src/iceberg/conversions.py

```diff
@@ -53,6 +53,13 @@
 from iceberg.utils.decimal import decimal_to_bytes, unscaled_to_decimal
 
 
+_bool_struct = Struct("<?")
```

Comment:
Do these count as constants? If so the python style guide recommends all caps for the name.

*PEP8 - Constants*
> Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL.

Reply:
Since they are private (starting with an underscore), they shouldn't be used outside of the file. We could also consolidate this later on with my open PR: https://github.com/apache/iceberg/pull/4920/files#diff-9747a5712c71edff07e52c56ae999d30fa9dde758ba94e376acd16819e055731R23-R27

Reply:
@samredai @Fokko correct me if I'm wrong. I consider this cached instance instead of constant because most of the time const is scalar value to me. These are class instance, kind of too complex object to consider it const by my understanding

Reply:
I would personally still think of it as a constant regardless of complexity, but I agree so long as they're not used outside of this file that's the more important thing to me.

Reply:
I changed it to all caps with leading underscore to follow the convention @Fokko @kbendick @samredai